src/domains/nodo-app/04_apim_nodo_fragment.tf

```diff
@@ -0,0 +1,25 @@
+# ndphost header
+resource "terraform_data" "sha256_ndphost_header" {
+  input = sha256(templatefile("./api_product/nodo_pagamenti_api/ndphost_header.xml.tpl", {
+    content = file("./api_product/nodo_pagamenti_api/env/${var.env}/ndphost_header.xml")
+  }))
+}
+resource "azapi_resource" "ndphost_header" {
```

Comment:
@aomegax can we use the policy fragment resource?
https://registry.terraform.io/providers/hashicorp/azurerm/3.116.0/docs/resources/api_management_policy_fragment